extensions/interactions/NumericInput/NumericInput.py

```diff
@@ -38,7 +38,16 @@ class NumericInput(base.BaseInteraction):
     can_have_solution = True
     show_generic_submit_button = True
 
-    _customization_arg_specs = []
+    _customization_arg_specs = [{
+        'name': 'inputGreaterThanZero',
```

Comment:
Please rename this to something like `inputGreaterThanOrEqualToZero`. Here and elsewhere.

Reply:
done